src/app/components/team/TeamReport/TeamReportComponent.js

```diff
@@ -104,6 +105,11 @@ const TeamReportComponent = ({ team, setFlashMessage }) => {
     });
   };
 
+  if (!team.smooch_bot) {
```

Comment:
This also requires smooch_bot to be requested in the team fragment. This time the query is on the parent component (in `TeamReport/index.js`)